src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -104,6 +104,10 @@ public enum OperationKind
         // Newly added
 
         ConditionalAccessExpression,
-        IncrementExpression
+        IncrementExpression,
+
+        Argument,
```

Comment:
@JohnHamby Now that we have `IArgument` as an `IOperation`, I think we can remove `OmittedArgumentExpression` and create a IArgument with kind==DefaultValue for optional argument in VB instead. This way it will be consistent with C# on argument list. Thoughts?